MicroServiceAppointment/Backend/Service/AppointmentService.cs

```diff
@@ -39,26 +42,33 @@ public class AppointmentService : IAppointmentService
             IPhysicianRepository physicianRepository,IProcedureTypeRepository procedureTypeRepository,
             IMedicineRepository medicineRepository)
         {
+            _medicineRepository = medicineRepository;
             _appointmentRepository = appointmentRepository;
             _physicianRepository = physicianRepository;
             _procedureTypeRepository = procedureTypeRepository;
-            _medicineRepository = medicineRepository;
         }
-
+        public AppointmentService(ISpecializationRepository specializationRepository,
+                IAppointmentRepository appointmentRepository, IPhysicianRepository physicianRepository,
+                IPatientRepository patientRepository, IAddressRepository addressRepository)
+        {
+            _appointmentRepository = appointmentRepository;
+            _physicianRepository = physicianRepository;
+            _specializationRepository = specializationRepository;
+            _addressRepository = addressRepository;
+            _patientRepository = patientRepository;
+        }
```

Comment:
I think you can only have one constructor if you're using dependency injection so delete the one with less parameters